core/src/main/java/com/linecorp/armeria/common/Cookie.java

```diff
@@ -68,11 +68,25 @@ static Cookie of(String name, String value) {
      *
      * @param name the name of the {@link Cookie}
      * @param value the value of the {@link Cookie}
+     *
+     * @deprecated Use {@link #secureBuilder(String, String)} instead to create secure cookie.
      */
+    @Deprecated
     static CookieBuilder builder(String name, String value) {
         return new CookieBuilder(name, value);
     }
 
+    /**
+     * Returns a newly created {@link CookieBuilder} which builds a {@link Cookie} with secure settings,
+     * which set "Strict" to "SameSite" attribute to never send cross-site requests.
```

Comment:
nit; The proposed javadoc has 2 clauses starting with `which` "which" doesn't sound natural 😄 

```
     * Returns a newly created {@link CookieBuilder} which builds a {@link Cookie} with secure settings.
     * {@code "Secure"}, {@code "HttpOnly"}, and {@code "SameSite=Strict"} are enabled by default.
```